genePred/genePred.go

```diff
@@ -119,8 +125,9 @@ func StringToIntSlice(text string) []int {
 	return answer
 }
 
-//TODO: May not work with - strand transcripts
-func CalcExonFrame(gene *GenePred) []int {
+// CaclExonFame calculates the frame of each exon in a genePred based on the information in the fields for exonStarts, exonStops and cdsStarts
+//May not work with - strand transcripts
```

Comment:
I would leave the TODO here, or make an issue for this.

Reply:
I'll make a TODO and maybe an issue as well